spec/models/password_disallows_frequent_changes_spec.rb

```diff
@@ -56,7 +56,7 @@
           'secure_password.password_disallows_frequent_changes.errors.messages.password_is_recent',
           timeframe: precise_distance_of_time_in_words(user.class.password_minimum_age)
         )
-        expect(user.errors.full_messages).to include error_string
+        expect(user.errors.full_messages).to include "Password #{error_string}"
```

Comment:
@tejanium what do you think about checking the `user.errors[:password]` presence here too? So we can follow the same pattern as you did below at `spec/models/password_disallows_frequent_reuse_spec.rb`

Reply:
Thanks for follow-up on it @tejanium 👍🏼